gcp/modules/locust/tasks/preferences.py

```diff
@@ -1,14 +1,16 @@
 from locust import HttpLocust, TaskSet, task
 import random
 
-class GpiiTasks(TaskSet):
-  _keys = ['carla', 'vladimir', 'wayne', 'omar', 'telugu']
+class PreferencesTasks(TaskSet):
+
+  _keys = ["carla", "vladimir", "wayne", "omar", "telugu"]
+
   @task
-  def prefByKey(self):
+  def getPrefByKey(self):
```

Comment:
Generally in Python, methods/functions should use `snake_case`. But maybe Locust dictates this name?

Reply:
Nope, Locust dictates nothing here, the only requirement is clear connection between task name and requests it is making, because task names are not present in stats. Let's switch to `snake_case` if you think it better follows Python naming style.